src/commands/project/list.js

```diff
@@ -0,0 +1,68 @@
+const { cli } = require('cli-ux')
+const { getProject } = require('../../requests/project')
+const { getResponseContent } = require('../../support/command/handle-response')
+const BaseCommand = require('../../support/command/base-command')
+
+class ListProjectCommand extends BaseCommand {
+    constructor(...props) {
+        super(...props)
+        this.logProject = this.logProject.bind(this)
+    }
+
+    async run() {
+        const { args } = this.parse(ListProjectCommand)
+        await this.listProjects(args['[OWNER]'])
+    }
+
+    async logProject(response) {
+        const responseObj = JSON.parse(await getResponseContent(response))
+        const projectCount = responseObj.projects.length
+        this.log(`Found ${ projectCount } project${ projectCount > 1 ? 's' : '' }.`)
```

Comment:
I wonder if we should count should be printed under the table (if at all).
It might be nice to only display it when there are no projects in the org, and then not display the table (just a suggestion).
`Found 0 project.` - this should be plural